File: resharper/tests.manual/tests.xunit/PropertyDataReferences/InvalidPropertyDataReference.cs
using System;
using Xunit;

namespace tests.xunit.eyeball.propertydata
{
    public class InvalidPropertyDataReference
    {
        // TEST: PropertyData attribute value should be marked invalid
        [Theory(Skip = "Invalid PropertyDataAttribute")]
        [MemberData("InvalidAccessibilityTheoryDataEnumerator")]
        public void UsingInvalidDataProperty(int value)
        {
            throw new InvalidOperationException("Should not run");
        }
    }
}

File: resharper/tests.manual/tests.xunit.vb/tests.xunit.eyeball.vb/PropertyDataReferences/InvalidPropertyDataReference.vb
Imports Xunit

Public Class InvalidPropertyDataReference
    
    ' TEST: PropertyData attribute value should be marked invalid
    <Theory(Skip:="Invalid PropertyDataAttribute")>
    <MemberData("InvalidAccessibilityTheoryDataEnumerator")>
    Public Sub UsingInvalidDataProperty(ByVal value As Int32)
        Throw New InvalidOperationException("Should not run")
    End Sub
End Class